flink-table/flink-sql-jdbc-driver/src/main/java/org/apache/flink/table/jdbc/BaseConnection.java

```diff
@@ -102,12 +101,6 @@ public int getTransactionIsolation() throws SQLException {
                 "FlinkConnection#getTransactionIsolation is not supported yet.");
     }
 
-    @Override
-    public SQLWarning getWarnings() throws SQLException {
```

Comment:
If these changes are not relevant with this issue, I'd suggest to move these to another commit (if they are trivial, we can just add a hotfix commit; if they are major changes, we should create a Jira and link to it)

Reply:
DONE

Reply:
I have move these commits to https://github.com/apache/flink/pull/22786